ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -140,7 +140,6 @@ objects:
         fi
 
         # set up cloud-provider-specific env vars
-        export KUBE_SSH_BASTION="$( oc --insecure-skip-tls-verify get node -l node-role.kubernetes.io/master -o 'jsonpath={.items[0].status.addresses[?(@.type=="ExternalIP")].address}' ):22"
```

Comment:
I see this in numerous other files in the repo. Is it correct there?

Reply:
Ya, I saw that too but we're not deploying a bastion in any other install IIUC. I could not find any other template with function to deploy bastion(`bastion_ssh`). If there is a need, let me know, I can add them but as part of separate PR which adds both function to deploy bastion and then update this variable.

Reply:
Currently installer template also contains two functions - `restore-cluster-state` and `recover-from-etcd-quorum-loss`, which are used to test DR scenarios. These tests are being run separately and on request. 

The scenarios are disruptive and require ssh bastion to be deployed on the hosts. Once the test is over the bastion is being removed and usual conformance tests are being run. Other e2e suites are not being affected by this and don't deploy ssh bastion.

Not sure what does `KUBE_SSH_BASTION` do, but its not related to `bastion-host` function and doesn't seem to affect DR scenarios.

Reply:
This is an environment variable that we can we use within e2e test context...

Reply:
okay, in this case it should not be set in DR functions